plugins/org.jboss.tools.openshift.cdk.server/src/org/jboss/tools/openshift/cdk/server/ui/internal/CDKServerWizardFragment.java

```diff
@@ -78,6 +80,7 @@ public Composite createComposite(Composite parent, IWizardHandle handle) {
 		credentials.addCredentialListener(new ICredentialCompositeListener() {
 			@Override
 			public void credentialsChanged() {
+				selectedUser = credentials.getUser();
```

Comment:
@adietish after the current line we get selectedUser updated after every credentials update which is then used in performFinish (line 210). And in line 123 it's just a one time assignment of selectedUser